src/main.cpp

```diff
@@ -634,12 +652,7 @@ void handleRoot() {
     const String countDown = FPSTR(count_down_script);
     rebootPage.replace("_TXT_M_REBOOT_", FPSTR(txt_m_reboot));
     sendWrappedHTML(rebootPage + countDown);
-    delay(500);
-#ifdef ESP32
-    ESP.restart();
-#else
-    ESP.reset();
```

Comment:
doesn't sound like `reset` should be used, it's not even [documented](https://arduino-esp8266.readthedocs.io/en/3.1.2/libraries.html#esp-specific-apis)